talks/de.json

```diff
@@ -1,126 +1,151 @@
 {
 	"respondTimeRemaining": {
 		"default": [
-			"Es sind noch {0} bis zu deinem {1}",
-			"Einen Moment bitte. Du hast noch {0} übrig auf dem {1}",
-			"Insgesamt hast du noch {0} auf dem {1}",
-			"Für den {1} verbleiben noch {0}. Keine Sorge, ich halet dich auf dem laufenden."
+			"Es gibt {0} Zeit links auf dieser {1}",
```

Comment:
It gives {0} time left (as left opposite of right) on this {1}

Reply:
will definately struggle with this line from homeassistant skill then ```Arrrr me matey. will do dat quicker than it takes ya to wonder why me just went all pirate on ya. Answer is. Cause i can "``` :)